stdlib/2/sets.pyi

```diff
@@ -0,0 +1,60 @@
+# Stubs for sets (Python 2)
+from typing import Any, Callable, Hashable, Iterable, Iterator, Mapping, Optional, TypeVar, Union
+
+_T = TypeVar('_T')
+Setlike = Union['BaseSet', Iterable]
+
+class BaseSet(Iterable[_T]):
+    def __init__(self) -> None: ...
+    def __len__(self) -> int: ...
+    def __repr__(self) -> str: ...
+    __str__ = ...  # type: Callable[[BaseSet[_T]], str]
+    def __iter__(self) -> Iterator[_T]: ...
+    def __cmp__(self, other: Any) -> None: ...
```

Comment:
This should return int by default, although the implementation always throws an exception.